eng/tox/install_dev_build_dependency.py

```diff
@@ -31,6 +32,23 @@
 
 logging.getLogger().setLevel(logging.INFO)
 
+
+def get_pip_command():
```

Comment:
Please move this to someplace within `azure-sdk-tools`. `functions.py` or `variables.py`? You've got this exact code copy/pasted three times or so.

Reply:
I _don't_ oppose the usage of this function though! 👍